ui/raidboss/data/03-hw/alliance/dun_scaith.js

```diff
@@ -342,31 +400,41 @@
     },
     {
       id: 'Dun Scaith Shadethrust',
-      regex: / 14:(?:1D23:Scathach|1C1A:Diabolos Hollow) starts using Shadethrust/,
+      regex: / 14:(?:1D23|1C1A):(Scathach|Diabolos Hollow) starts using Shadethrust/,
```

Comment:
`(Scathach|Diabolos Hollow)` should be non-capturing.

Reply:
It is now a non matching group